README.md

```diff
@@ -19,5 +19,5 @@ We welcome pull requests from the community.
 Use the following command to run the tests:
```

Comment:
Use the following command to run the tests: -->
To run the tests, execute the following command from within the root directory of the project: